server/src/DBService/InvestigationInfo/Mutation.ts

```diff
@@ -73,7 +73,7 @@ mutation deleteinvestigationReasonId ($epidemiologyNumberInput: Int!, $oldComple
 `;
 
 export const UPDATE_INVESTIGATION_STATIC_INFO = gql`
-mutation updateInvestigationStaticInfo ($fullNameInput: String, $identificationTypeInput: String, $identityNumberInput: String, $epidemiologyNumberInput: Int!) {
+mutation updateInvestigationStaticInfo ($fullNameInput: String, $identificationTypeInput: Int, $identityNumberInput: String, $epidemiologyNumberInput: Int!) {
```

Comment:
can it be null? if it can't write `Int!`

Reply:
yes it can be null